1-Draft/RFCXXXX-RFC-Invoke-DscResource.md

```diff
@@ -0,0 +1,126 @@
+---
+RFC: RFCXXXX
+Author: Gael Colas
+Status: Draft
+SupersededBy: N/A
+Version: 0.6
+Area: Microsoft.PowerShell.DesiredStateConfiguration
+---
+
+# Invoke-DscResource
+
+Add cross-platform support for `Invoke-DscResource` in PowerShell 7+ without dependency on LCM and WMI.
+
+This RFC addresses the need to leverage the DSC ecosystem of resources from newer versions of PowerShell, the way to decouple the execution of resources from the LCM and CIM/WMI, and the user experience from a consumer and solution vendor/integrator point of view.
+
+## Motivation
+
+1. DSC has been cited by users and solution partners as a blocker for moving from Windows PowerShell to PowerShell 7+.
+
+2. As part of the latest survey on DSC usage started in June 2019, the top requested feature was support for Invoke-DscResource in PSCore without WMI dependency.
+
+3. The DSC Ecosystem and Community would greatly benefit from enabling DSC resources to be used in imperative scripts, in current user context.
+
```

Comment:
So point 1 is meant to include them, but I thought name dropping would feel limiting because maybe vmware, CFEngine or the ISVs could also leverage this work...
Your call...